subsys/bluetooth/mesh/Kconfig

```diff
@@ -301,6 +301,26 @@ config BT_MESH_ADV_EXT
 
 endchoice
 
+if BT_MESH_ADV_EXT
+
+config BT_MESH_RELAY_ADV_SETS
+	int "Maximum of simultaneous relay message support"
+	default 0
+	range 0 BT_EXT_ADV_MAX_ADV_SET
+	depends on BT_MESH_RELAY
+	help
+	  Maximum of simultaneous relay message support. Requires controller support
+	  multiple advertising sets.
+
+config BT_MESH_GATT_ADV
```

Comment:
How does GATT provisioning work in adv_multi.c without this? If it's possible to set this to `n` while `GATT_PROXY` is `y`, we have to be able to send proxy beacons on the main advertiser